### ISSUE TITLE
Last Facebook SDK for Unity update is Jun 8, 2021, there are numerous issues with it by now especially with iOS 15. Can you merge changes from your other sdks here already?

### ISSUE THREAD

**Issue by NikolayTimofeev** [facebook/facebook-sdk-for-unity#590]
## Checklist

- [ ] I've updated to the latest released version of the SDK
- [ ] I've searched for existing feature requests on [GitHub issues](https://github.com/facebook/facebook-sdk-for-unity/issues)
- [ ] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [ ] I've given my issue the title: `Feature Request: [name of my feature request]`

## Goals

What do you want to achieve?

## Expected Results

What do you expect to happen?

## Code Samples & Details

Please provide a code sample, as well as any additional details, to help us track down the issue. If you can provide a link to a test project that allows us to reproduce the issue, this helps us immensely in both the speed and quality of the fix.

**Note:** Remember to format your code for readability:

```swift
// INSERT YOUR CODE HERE

var example = "Example code"
```

**Comment by Goncharuk-Nikita** [comment]
+1 
We are losing money because of this.

**Comment by Gamenator** [comment]
Please answer, are you going to update sdk for unity?

**Comment by Goncharuk-Nikita** [comment]
Any updates?

**Comment by NikolayTimofeev** [comment]
@Goncharuk-Nikita nope, last sdk update from Facebook is still 11.0 from 8 of June 2021, more than half a year at this point 🤦‍♂️
Facebook is a dead company I guess.

**Comment by sandolkakos** [comment]
Any update on that situation?
Our ANR and Crashes on GooglePlay show errors coming from the `FB for Unity` plugin that are already fixed on the FB Android SDK.

please update the SDK for Unity, we are losing players because of the crashes, and also the stores, from time to time, are not approving our weekly new app versions when they get the crashes while reviewing it.

**Comment by adiohaion** [comment]
I'm joining the requests here. Please release a new Unity version, it doesn't make sense not to release an update in 6 months for millions of users using games and apps built in Unity.

**Comment by LingXin94** [comment]
We need update!

**Comment by DulgiKim** [comment]
When I tried to make v12 FBSDK with project of this repository, I knew there are some 'missing' interfaces in IOS plugin.
In FacebookGamingServices.framework, there exists only FBSDKChooseContextDialog interface.
But Xcode build requires FBSDKCreateContextDialog & FBSDKSwitchContextDialog, too.
Is there any problem with updating pod packages?

**Comment by unity16th30317** [comment]
Please Update Unity Facebook SDK to v.13!
Because this 
[https://developers.facebook.com/blog/post/2022/02/24/introducing-facebook-platform-sdk-13-related-updates/](url)
seems like facebook not supported v.11 so please update!

**Comment by Cicalinarrot** [comment]
I can't even get events to work, has anyone solved this?

**Comment by owaisfb** [comment]
SDK package for v12 is now available on the download page. We've been resource constrained the past months but we will have renewed support for the SDK going forward. Thank you for your patience!
We will be v13 releasing soon (ETA is this week)
